src/utils/sentry.ts

```diff
@@ -15,11 +15,14 @@ Sentry.configureScope((scope) => {
   scope.setTag("Node environment", process.env.NODE_ENV);
 });
 
-export const configureReq = (
+export const addRequestScopeToSentry = (
   request: NextApiRequest | IncomingMessage,
 ): void => {
   const headers = Object.entries(request.headers).reduce(
-    (acc, [key, value]) => (key !== "cookie" ? { ...acc, [key]: value } : acc),
+    (acc, [key, value]) =>
+      key.toLowerCase() !== ("cookie" || "authorization")
```

Comment:
Wait, it should have changed on Friday.

Reply:
Nvm, I miss-read your comment. Do you mean that `!== ("cookie" || "authorization")` won't work?

Reply:
I couldn't reproduce a case where an `Authroization` field was provided.

I read about this and the only thing I can think of is casing, or prefix (i.e. `WWW` or `PROXY`).

Reply:
Here's what I mean

```
>  "cookie" !== ("cookie" || "authorization")
false
> "other" !== ("cookie" || "authorization")
true
>  "authorization" !== ("cookie" || "authorization")
true
> ("cookie" || "authorization")
'cookie'
```

It wouldn't work in the case of `authorization` since your `||` will always return `cookie`

Reply:
Damn, nice catch. Fixed.